hibernate-redis/src/main/java/org/hibernate/cache/redis/regions/RedisDataRegion.java

```diff
@@ -179,4 +190,8 @@ public long nextTimestamp() {
     public int getTimeout() {
         return cacheLockTimeout;
     }
+
+    public RedisAccessStrategyFactory getAccessStrategyFactory() {
```

Comment:
let's revisit visibility in a follow up pull request before making next release.